test/recipe-test.js

```diff
@@ -1,15 +1,18 @@
 import { expect } from 'chai';
 
 import Recipe from '../src/recipe';
-import data from '../src/data/recipe-data';
+import recipeData from '../src/data/recipe-data';
+import ingredientData from '../src/data/ingredient-data'
 
 describe('Recipe', function() {
   let recipe;
   let recipeInfo;
+  let ingredientInfo;
 
   beforeEach(function() {
-    recipeInfo = data.recipeData[0];
-    recipe = new Recipe(recipeInfo);
+    recipeInfo = recipeData[0];
+    ingredientInfo = ingredientData
+    recipe = new Recipe(recipeInfo, ingredientInfo);
```

Comment:
Similar to the corresponding recipe class, We may need to review my idea of changing the class per the specs of the original project.  I think its working perfectly, as it is though.